index.html

```diff
@@ -37,7 +37,7 @@
   <body>
     <section id='abstract'>
       <p>
-        The new input event type introduced in this document provide web developers a way to listen to "click" action of non-primary buttons and also prevent the default behaviour on those events like opening a new tab when middle clicking on a link. This new input type is needed because the <a href="https://www.w3.org/TR/DOM-Level-3-Events/#event-type-click">click</a> event is restricted to primary button only.
+        The new input event type introduced in this document provides web developers a way to listen to the "click" action of non-primary buttons, and potentially prevent their default action (eg. opening a new tab when middle clicking on a link). This new input type is needed because the <a href="https://www.w3.org/TR/DOM-Level-3-Events/#event-type-click">click</a> event is restricted to primary button only.
```

Comment:
I thought the formal term was "default action", but it looks like that's [no longer the case](https://dom.spec.whatwg.org/#action-versus-occurance).  I'll reword this to avoid "action" and instead talk about "canceling".